src/lib/cht-session.ts

```diff
@@ -44,25 +43,30 @@ export default class ChtSession {
     }
     
     const userDetails = await ChtSession.fetchUserDetails(authInfo, username, sessionToken);
-    const facilityId = userDetails.isAdmin ? ADMIN_FACILITY_ID : userDetails.facilityId;
-    if (!facilityId) {
+    const facilityIds = userDetails.isAdmin ? [ADMIN_FACILITY_ID] : userDetails.facilityId;
+    if (!facilityIds || facilityIds?.length === 0) {
       throw Error(`User ${username} does not have a facility_id connected to their user doc`);
     }
     
-    return new ChtSession(authInfo, sessionToken, username, facilityId);
+    return new ChtSession(authInfo, sessionToken, username, facilityIds);
   }
 
   public static createFromDataString(data: string): ChtSession {
-    const parsed:any = JSON.parse(data);
-    return new ChtSession(parsed.authInfo, parsed.sessionToken, parsed.username, parsed.facilityId);
+    const parsed: { 
+      authInfo: AuthenticationInfo;
+      sessionToken: string;
+      username: string;
+      facilityIds: string[]; 
+    } = JSON.parse(data);
+    return new ChtSession(parsed.authInfo, parsed.sessionToken, parsed.username, parsed.facilityIds);
```

Comment:
isn't it `facilityId` on old versions of core?  

seems we need two tests for this PR. One with each of the values returned by cht-core

Reply:
don't we need to invalidate all existing sessions? since you're changing the values which are stored? anybody who is logged in will fail this code path

Reply:
I think this Is something that has to be done via kubectl https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl/#edit-secret. Not sure if there's a way to regenerate it without deleting then creating it again